diff --git a/dist/index.js b/dist/index.js
index c62735ec73824f46db3c85ec78e2007fab672fe8..09ee61398d7f591892aaccf010513fa6302e7a17 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,8 +1,4 @@

 'use strict'

-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./didyoumean3.cjs.production.min.js')
-} else {
-  module.exports = require('./didyoumean3.cjs.development.js')
-}
+module.exports = require('./didyoumean3.cjs.production.min.js')
